frontend/src/components/panels/Document.vue

```diff
@@ -123,7 +123,7 @@
 						<CanvasRuler :origin="rulerOrigin.y" :majorMarkSpacing="rulerSpacing" :numberInterval="rulerInterval" :direction="'Vertical'" />
 					</LayoutCol>
 					<LayoutCol :class="'canvas-area'">
-						<div class="canvas" ref="canvas">
+						<div class="canvas" ref="canvas" :style="{ cursor: canvasCursor }" v-on:pointerdown="canvasPointerDown">
```

Comment:
`@pointerdown="(e: PointerEvent) => canvasPointerDown(e)"`

Reply:
What's wrong with `@pointerdown="canvasPointerDown"`?

Reply:
We're being explicit with our type usage by convention.